diff -urN gmpc-last.fm-0.15.5.0/src/Makefile.am gmpc-last.fm-0.15.5.0.new/src/Makefile.am
--- gmpc-last.fm-0.15.5.0/src/Makefile.am	2007-11-29 14:54:40.000000000 +0100
+++ gmpc-last.fm-0.15.5.0.new/src/Makefile.am	2008-02-04 10:30:28.476614279 +0100
@@ -1,6 +1,6 @@
-libdir=${prefix}/share/gmpc/plugins/
+pluginsdir=$(libdir)/gmpc/
 
-lib_LTLIBRARIES = lastfmplugin.la
+plugins_LTLIBRARIES = lastfmplugin.la
 
 lastfmplugin_la_SOURCES=	\
 	plugin.c 
